scripting/SurfTimer-discord.sp

```diff
@@ -7,6 +7,7 @@
 #include <mapchallenge>
 #pragma newdecls required
 #pragma semicolon 1
+// #define DEBUG // Enable this line and compile again to enable DEBUG messages
 
 public Plugin myinfo =
 {
```

Comment:
Can you please also change the version to 2.5.7 so that I can trigger the build? Thanks!

Reply:
aye aye